src/containers/Tenant/ObjectSummary/ObjectSummary.tsx

```diff
@@ -276,6 +279,7 @@ function ObjectSummary(props: ObjectSummaryProps) {
     const onOpenPreview = () => {
         dispatch(setShowPreview(true));
         dispatch(setTopLevelTab(TENANT_GENERAL_TABS_IDS.query));
+        dispatch(setQueryTab(TENANT_QUERY_TABS_ID.newQuery));
```

Comment:
Technically, table preview doesn't depend on the upper section content. Do you think it is required to force change tab to new query?

Reply:
It's rendered in query result section. Though it doesn't depend on query editor content, it seems more suitable there, I don't thinks we should mess it with QueryTables

Reply:
The case is: "I am watching queries history and I want to match it with particular table preview."
In the previous interface version it was possible. I think this should be discussed with PM, because I don't know frequency and importance of this case.

Reply:
I still don't understand this case. For the table columns, their name and types there is schema under the tree. Preview is mostly for the table content. I don't see an issue where there is a need to match table content and particular queries within one tab. 

Additionally, discussed it with PM. It's seems, that this case isn't important